### PR TITLE
WIP: Do the 10k

### PR DESCRIPTION
**Pull request by wasade**
This is a work in progress that refactors path management, and includes support for generating many other datasets of interest for downstream use.

Path management is not complete yet as, in the ideal case, there is a data access object to facilitate developers (e.g., `AGData.bdiv('notrim', '1k', 'fecal')` to get back the corresponding `DistanceMatrix`). In addition, the shear number of paths tracked is large but the patterns suggest it should be possible to not explicitly define them. However, right now, development is on a near path of least resistance.

### CODE REVIEW

**Comment by wasade** [comment]
...Prior to asking for review, I need to pull in the changes necessary for compute on the dataset as a whole and get those changes back in sync.

File: ipynb/primary-processing/03-pick_otus.md [hunk 1]
```diff
@@ -7,25 +7,11 @@ Pick OTUs (for full and trimmed data) at approximately genus level resolution (9
...
-
-Before we go too far, let's make sure the files we need are present.
-
-```python
->>> filtered_sequences       = agu.get_existing_path(agenv.paths['filtered-sequences'])
->>> filtered_sequences_100nt = agu.get_existing_path(agenv.paths['filtered-sequences-100nt'])
+>>> chp_path = agenv.activate('03-otus')
 ```
 
 And, let's make sure that the output files we need do not already exist.
```
Comment (JWDebelius) [inline ipynb/primary-processing/03-pick_otus.md]: Should there be code after this line?

**Comment by JWDebelius** [comment]
Is it approaching the point where this can be cleaned up for review?

**Comment by wasade** [comment]
Should be basically there

**Comment by wasade** [comment]
Lit up green

**Comment by JWDebelius** [comment]
Sweet. The only thing I couldn't get to work when I ran through this branch was associated with the click command, and processing subsequent to the click commands. Given that this PR didn't touch that, I'm inclined that, other than the one comment, about text, I think it should be good.

**Comment by JWDebelius** [comment]
Thanks!

**Pull request merged**